dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/BrowserAPI.java

```diff
@@ -254,256 +260,121 @@ public WfData(Contentlet contentlet, List<Integer> permissions, User user, boole
-                    }
-				}
-			} catch (Exception e1) {
-				Logger.error(this, "Could not load HTMLPages : ", e1);
-			}
-
-			for (IHTMLPage page : pages) {
-				
-				boolean isContentlet = page instanceof Contentlet;
-				// include only pages for the given language
-				if(isContentlet && languageId > 0 && ((Contentlet)page).getLanguageId()!= languageId)
-					continue;
-				
-				List<Integer> permissions = new ArrayList<Integer>();
-				try {
-					permissions = permissionAPI.getPermissionIdsFromRoles(page, roles, user);
-				} catch (DotDataException e) {
-					Logger.error(this, "Could not load permissions : ", e);
-				}
-				if (permissions.contains(PERMISSION_READ)) {
-				    WfData wfdata = page instanceof Contentlet ? new WfData((Contentlet)page, permissions, user, showArchived) : null;
-				    
-				    if(wfdata!=null && wfdata.skip)
-				        continue;
-				    
-					Map<String, Object> pageMap = page.getMap();
-					pageMap.put("mimeType", "application/dotpage");
-					pageMap.put("permissions", permissions);
-					pageMap.put("name", page.getPageUrl());
-					pageMap.put("description", page.getFriendlyName());
-					pageMap.put("extension", "page");
-					pageMap.put("isContentlet", isContentlet);
-					
-					if(wfdata!=null ) {
-    		            pageMap.put("wfActionMapList", wfdata.wfActionMapList);
-    	                pageMap.put("contentEditable", wfdata.contentEditable);
-					}
-					
-					if(isContentlet) {
-					    pageMap.put("identifier", page.getIdentifier());
-		                pageMap.put("inode", page.getInode());
-					    pageMap.put("languageId", ((Contentlet)page).getLanguageId());
-					    
-					    Language lang = APILocator.getLanguageAPI().getLanguage(((Contentlet)page).getLanguageId());
-					    
-					    pageMap.put("languageCode", lang.getLanguageCode());
-		                pageMap.put("countryCode", lang.getCountryCode());
-		                pageMap.put("isLocked", page.isLocked());
-		                pageMap.put("languageFlag", LanguageUtil.getLiteralLocale(lang.getLanguageCode(), lang.getCountryCode()));
-		            
-		              
-					}
-
-                    pageMap.put("hasLiveVersion", APILocator.getVersionableAPI().hasLiveVersion(page));
-					pageMap.put("statusIcons", UtilHTML.getStatusIcons(page));
-					pageMap.put("hasTitleImage",String.valueOf(((Contentlet)page).getTitleImage().isPresent()));
-					pageMap.put("__icon__", "pageIcon");
-					returnList.add(pageMap);
-				}
-			}
+            if (parent != null) {
+                if (showWorking) {
+                    links.addAll(folderAPI.getLinks(parent, true, false, user, false));
+                } else {
+                    links.addAll(folderAPI.getLiveLinks(parent, user, false));
+                }
+                if (showArchived)
+                    links.addAll(folderAPI.getLinks(parent, true, showArchived, user, false));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)